apps/deploy-web/src/services/analytics/analytics.service.ts

```diff
@@ -146,7 +146,7 @@ export class AnalyticsService {
 
     for (const key in user) {
       if (key !== "id") {
-        event.set(AMPLITUDE_USER_PROPERTIES_MAP[key] || AMPLITUDE_USER_PROPERTIES_MAP, user[key]);
+        event.set(AMPLITUDE_USER_PROPERTIES_MAP[key as keyof AnalyticsUser] || key, String(user[key as keyof typeof user]));
```

Comment:
@ygrishajev I need your attention here. Since I couldn't understand purpose of the code, and it doesn't match `event.set` types, I assumed there is a bug

Reply:
ur change looks correct to me